Documentation/ceph-upgrade.md

```diff
@@ -57,6 +57,12 @@ those releases.
   easier to discover for new Rook users. `cluster/examples/kubernetes/ceph` manifests can now be
   found in `deploy/examples`.
 
+* The admission controller has been integrated inside the operator. So it is not running as
+  a separate deployment anymore. This means that the webhook server certificates are now stored in
+  the operator. The operator manifest must be updated to use the one provided in
+  `deploy/examples/operator.yaml`. If you are using Helm to manage the deployment there is nothing
+  to do.
+
```

Comment:
When do users need to make this change? It would be helpful to put it in the upgrade steps. Would that be a step before `1. Update common resources and CRDs`? Or perhaps after?